### DISCUSSION TITLE
Downloaded without consent

### DISCUSSION THREAD

**Discussion by yxkimxrax** [Tyrrrz/DiscordChatExporter#596] [Q&A]
Hello! This isn't an attack on the creators as I think this is a very cool little app! But this was downloaded as a watchdog-like thing onto my laptop to track my conversations. I wanted to know the easiest ways to delete this and if there are any files that may be hidden around my desktop that I also need to delete. Thanks!

**Comment by Tyrrrz** [comment]
It's hard to say without much details, but generally speaking deleting the program from your computer should solve the problem. Try searching for `DiscordChatExporter.Core.dll` file on your computer.

**Comment by 96-LB** [comment]
If someone is using your token to export your chats, that could be a problem in itself. Besides deleting the program files (which are usually all in one place), you may also want to change your Discord password so that your token is regenerated.
